crates/rome_js_syntax/src/expr_ext.rs

```diff
@@ -379,6 +418,10 @@ impl JsObjectExpression {
     pub fn has_trailing_comma(&self) -> bool {
         self.members().trailing_separator().is_some()
     }
+
+    pub fn is_empty(&self) -> bool {
+        self.members().len() == 0
```

Comment:
I think this should work
```suggestion
        self.members().is_empty
```

Reply:
👍👍👍 Thx!!!